packages/client/wallets/smart-wallet/src/api/CrossmintWalletService.ts

```diff
@@ -21,7 +20,13 @@ export class CrossmintWalletService extends BaseCrossmintService {
     async getSmartWalletConfig(
         user: UserParams,
         chain: EVMBlockchainIncludingTestnet
-    ): Promise<{ kernelVersion: string; entryPointVersion: string; signers: { signerData: SignerData }[] }> {
+    ): Promise<{
+        kernelVersion: string;
+        entryPointVersion: string;
+        userId: string;
+        signers: { signerData: SignerData }[];
+        smartContractWalletAddress?: string;
```

Comment:
I've added the two new properties here in [this PR](https://github.com/Paella-Labs/crossbit-main/pull/14114).